packages/docusaurus-theme-search/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "@markprompt/docusaurus-theme-search",
```

Would this be enough? Or do you suspect we might be releasing other Docusaurus plugins in the future? 

```suggestion
  "name": "@markprompt/docusaurus",
```

---

Ah no I see the naming conventions now at https://docusaurus.io/docs/using-plugins#module-shorthands. This is as it should be 👌

---

In fact, I started using the name you proposed.